bitnami/discourse/3/debian-11/rootfs/opt/bitnami/scripts/libdiscourse.sh

```diff
@@ -267,6 +267,8 @@ discourse_create_conf_file() {
         discourse_conf_set "smtp_password" "$DISCOURSE_SMTP_PASSWORD"
         discourse_conf_set "smtp_enable_start_tls" "$([[ "$DISCOURSE_SMTP_PROTOCOL" = "tls" ]] && echo "true" || echo "false")"
         discourse_conf_set "smtp_authentication" "$DISCOURSE_SMTP_AUTH"
+        discourse_conf_set "smtp_open_timeout" "$DISCOURSE_SMTP_OPEN_TIMEOUT"
+        discourse_conf_set "smtp_read_timeout" "$DISCOURSE_SMTP_READ_TIMEOUT"
```

Comment:
To not to overwrite the default value in the conf file in case you do not set these env vars, please apply this suggestion.

```suggestion
        ! is_empty_value "$DISCOURSE_SMTP_OPEN_TIMEOUT" && discourse_conf_set "smtp_open_timeout" "$DISCOURSE_SMTP_OPEN_TIMEOUT"
        ! is_empty_value "$DISCOURSE_SMTP_READ_TIMEOUT" && discourse_conf_set "smtp_read_timeout" "$DISCOURSE_SMTP_READ_TIMEOUT"
```